transport-classes-io_uring/src/main/java/io/netty/channel/uring/IoUringBufferRing.java

```diff
@@ -74,28 +78,42 @@ boolean isUsable() {
     }
 
     void initialize() {
-        fillBuffers();
+        for (short i = 0; i < batchSize; i++) {
+            addBuffer(i);
+            numBuffers++;
+            lastGeneratedBid = i;
+        }
+        assert numBuffers % 2 == 0;
```

Comment:
This assert should be covered by the `assert batchSize % 2 == 0;` in the constructor.

Reply:
yeah but can't harm to also have it here imho